infrastructure-scripts/claude_start.sh

```
@@ -429,8 +591,8 @@ if [ -n "$MODE" ]; then
             if [ ! -f "$API_PROXY_PATH" ]; then
                 echo -e "${RED}❌ LLM self-hosting repository not found${NC}"
                 echo -e "${BLUE}💡 LLM proxy (for Qwen and Cerebras) is maintained in the separate claude_llm_proxy repository${NC}"
-                echo -e "${BLUE}💡 Clone it with: cd ~/projects && git clone https://github.com/jleechanorg/claude_llm_proxy.git${NC}"
-                echo -e "${BLUE}💡 Repository: https://github.com/jleechanorg/claude_llm_proxy${NC}"
+                echo -e "${BLUE}💡 Clone it with: cd ~/projects && git clone $(git config --get remote.origin.url)
```

Shellcheck (0.10.0)

[warning] Did you forget to close this double quoted string?

(SC1078)